### PR TITLE
:sparkles: Change copy texts and style of library dialogs

### PR DESCRIPTION
**Pull request by hirunatan**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: frontend/resources/locales.json [hunk 1]
```diff
@@ -1269,13 +1263,13 @@
...
     "translations" : {
-      "en" : "Are you sure you wan't to delete this project?"
+      "en" : "Are you sure you want to delete this project?"
     }
   },
   "modals.delete-project-confirm.title" : {
     "used-in" : [ "src/app/main/ui/dashboard/files.cljs:56" ],
     "translations" : {
-      "en" : "Deleting project"
+      "en" : "Delete project"
```
Comment (niwinz) [inline frontend/resources/locales.json:1272]: Why this change? The design/prototypes expectes the `Deleting project` title.

File: frontend/src/app/main/ui/dashboard/grid.cljs [hunk 2]
```diff
@@ -109,10 +109,12 @@
            (dom/stop-propagation event)
            (st/emit! (modal/show
                       {:type :confirm
-                       :message (t locale "modals.add-shared-confirm.message" (:name file))
-                       :title (t locale "modals.add-shared-confirm.title")
+                       :message ""
+                       :title (t locale "modals.add-shared-confirm.message" (:name file))
                        :hint (t locale "modals.add-shared-confirm.hint")
+                       :cancel-label ""
```
Comment (niwinz) [inline frontend/src/app/main/ui/dashboard/grid.cljs]: why not just omit the :cancel-label in this case? or use `nil` instead of `""`

File: frontend/resources/locales.json (hunk 1 above)
Comment (hirunatan) [reply frontend/resources/locales.json:1272]: It's asked for in this issue https://tree.taiga.io/project/uxboxproject/issue/759

File: frontend/src/app/main/ui/dashboard/grid.cljs (hunk 2 above)
Comment (hirunatan) [reply frontend/src/app/main/ui/dashboard/grid.cljs]: If you omit cancel-label, then the default "Cancel" label is used (see confirm.cljs). We can use other system, but there should be a way to distinguish between "the default label" or "no label and no button".

Comment (hirunatan) [reply frontend/src/app/main/ui/dashboard/grid.cljs]: See https://tree.taiga.io/project/uxboxproject/issue/760

File: frontend/resources/locales.json (hunk 1 above)
Comment (niwinz) [reply frontend/resources/locales.json:1272]: I think, they should first agree before fill issues, the prototypes says "Deleting projects". But is ok.

File: frontend/src/app/main/ui/dashboard/grid.cljs (hunk 2 above)
Comment (niwinz) [reply frontend/src/app/main/ui/dashboard/grid.cljs]: Hmm in case we want to have the ability to omit cancel button, i think is better have `:default` value for default and omit it by default. Empty string is very confusing as option for ommit it.

Comment (hirunatan) [reply frontend/src/app/main/ui/dashboard/grid.cljs]: fixed!
